extra-network/netplan/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.98
+# SRCTBL="https://github.com/CanonicalLtd/netplan/archive/$VER.tar.gz"
```

Comment:
What is with this?

Reply:
At the time I committed that, CanonicalLtd's version FTBFS, so a fork was used instead. In the next commit this will be uncommented.